contracts/USM.sol

```diff
@@ -131,4 +148,75 @@ contract USM is BufferedToken {
         latestFumPrice = fumPrice();
         emit LatestFumPriceChanged(previous, latestFumPrice);
     }
+
+    /** INTERNAL FUNCTIONS */
+
+    /**
+     * @notice Funds the pool with ETH, minting FUM at its current price
+     */
+    function _fund(address to, uint ethIn) internal {
+        uint _fumPrice = fumPrice();
+        uint fumOut = ethIn.wadDiv(_fumPrice);
+        ethPool = ethPool.add(ethIn);
+        fum.mint(to, fumOut);
+        _setLatestFumPrice();
+    }
+
+    /**
+     * @notice Mint USM from Eth. Inheriting contract must handle the Ether transfers.
+     */
+    function _mint(uint ethAmount) internal returns (uint) {
```

Comment:
Once you remove `ethPool`, this function is reduced to `_mint(msg.sender, _ethToUsm(ethAmount));` and can be merged with `mint`.

Reply:
Agreed (and generally with @albertocuestacanada's comments here)